src/plugins/jsx/fromCodePoint.js

```diff
@@ -40,18 +40,20 @@ if (!fromCodePoint) {
       if (
         !isFinite(codePoint) || // `NaN`, `+Infinity`, or `-Infinity`
         codePoint < 0 || // not a valid Unicode code point
-        codePoint > 0x10FFFF || // not a valid Unicode code point
+        codePoint > 0x10ffff || // not a valid Unicode code point
         floor(codePoint) != codePoint // not an integer
       ) {
         throw RangeError("Invalid code point: " + codePoint);
       }
-      if (codePoint <= 0xFFFF) { // BMP code point
+      if (codePoint <= 0xffff) {
+        // BMP code point
         codeUnits.push(codePoint);
-      } else { // Astral code point; split in surrogate halves
+      } else {
+        // Astral code point; split in surrogate halves
         // https://mathiasbynens.be/notes/javascript-encoding#surrogate-formulae
         codePoint -= 0x10000;
-        highSurrogate = (codePoint >> 10) + 0xD800;
```

Comment:
It's always so interesting how little things trigger knee-jerk reactions. I always prefer capitalized hex haha.